src/pages/trade-types/multiplier/index.tsx

```diff
@@ -27,6 +27,22 @@ const StyledHeader = styled(Header)`
 `
 
 const Multipliers = () => {
+    const target = '#hero-section'
```

Comment:
Could u pls move `target`, `options`, `lazy_components` out of the Multipliers :pray: